src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProvider.cs

```diff
@@ -83,21 +104,78 @@ void IDkmClrResultProvider.GetChildren(DkmEvaluationResult evaluationResult, Dkm
                 return;
             }
 
-            var stackFrame = evaluationResult.StackFrame;
-            GetChildrenAndContinue(dataItem, workList, stackFrame, initialRequestSize, inspectionContext, completionRoutine);
+            var expansion = dataItem.Expansion;
+            if (expansion == null)
+            {
+                var enumContext = DkmEvaluationResultEnumContext.Create(0, evaluationResult.StackFrame, inspectionContext, new EnumContextDataItem(evaluationResult));
+                completionRoutine(new DkmGetChildrenAsyncResult(new DkmEvaluationResult[0], enumContext));
```

Comment:
I'm assuming `Array.Empty` is not available here, but maybe we want to introduce our own constant?